target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -220,7 +220,8 @@ case "$board" in
 		hw_mac_addr="$(mtd_get_mac_binary product-info 0x8)"
 		macaddr_add "$hw_mac_addr" "$PHYNBR" > "/sys${DEVPATH}/macaddress"
 		;;
-	tplink,ex220-v1)
+	tplink,ex220-v1|\
```

Comment:
it's possible to do this in dts. See   https://github.com/openwrt/mt76/commit/65cc3daf2a332cc658e9f7438cdadde4392e672e